index.js

```diff
@@ -14,11 +17,21 @@ module.exports = {
     this._ensureThisImport();
 
     this.import('vendor/ember-simple-auth/register-version.js');
+    this.import('vendor/base64.js');
   },
 
-  treeForVendor: function() {
-    var content = 'Ember.libraries.register(\'Ember Simple Auth\', \'' + version + '\');';
-    return writeFile('ember-simple-auth/register-version.js', content);
+  treeForVendor(vendorTree) {
+    var content =
+      "Ember.libraries.register('Ember Simple Auth', '" + version + "');";
+    var registerVersionTree = writeFile(
+      'ember-simple-auth/register-version.js',
+      content
+    );
+    var base64Tree = new Funnel(path.dirname(require.resolve('base-64')), {
+      files: ['base64.js']
+    });
+
+    return MergeTrees([vendorTree, registerVersionTree, base64Tree]);
```

Comment:
since you don't have a `vendor` folder you could skip the `vendorTree` here, but then again once you add a folder you might get surprised in the future when it would not work if you removed it here 😉